content/operator/gb/index.en.md

```diff
@@ -0,0 +1,12 @@
+---
+draft: false
+title: "National Rail"
+description: "Informationen über die FIP-Bedingungen bei National Rail."
+country:
+  - "united_kingdom"
+operator: "gb"
```

Comment:
```suggestion
operator: "national-rail"
```

National Rail would be the correct operator name, we should also reflect this in the ID.

Reply:
No I don't agree with this. In the ID we usually use the same like the apprevation on the FIP Coupon. Like SBB for SBB CFF FFS or ZSR instead of ZSSK / ZSR. "GB" is the official apprevation for National Rail and also written like this on the FIP Coupon.

Reply:
Ok, fair, but shouldn't we mention it in the name as well then?

Reply:
Yes, or "GB (National Rail)" / "National Rail (GB)".

Reply:
Hmm to be honest I think it's one think to use "gb" ad the id (because of the apprevation on the FIP Coupon), but _National Rail_ ist the official complete name. https://en.wikipedia.org/wiki/National_Rail
So I wouldn't like to make it more complicated :) I hope this is fine for you 🙊